packages/render-fidelity-tools/test/config.json

```diff
@@ -243,7 +249,8 @@
         "radius": 3
       },
       "exclude": [
-        "dspbr-pt"
+        "dspbr-pt",
+        "stellar"
```

Comment:
What's the trouble with Suzanne?

Reply:
Sorry, as a starting point, I mostly added exludes based on the dspbr-pt flags. Just let me know of anything else is missing. 
Added Suzanne